### ISSUE TITLE
Is there any way to create multiFile appender Daily Rolling instead of making it roll on the size?

### ISSUE THREAD

**Issue by AgnibhaBose** [log4js-node/log4js-node#645]
I have an scenario where I have to use multiFile appender but the rolling strategy that I can found is only based on the size but in my case I want it to be like dateFile. Is there any way to do so?

**Comment by AgnibhaBose** [comment]
Currently I am doing it in a hackish way.

First 


```
const log4js = require('log4js');
log4js.configure({
    appenders: {
        everything: {
            type: 'multiFile', base: './', property: 'hack', extension: '.log'
        }
    },
    categories: {
        default: { appenders: ['everything'], level: 'debug' }
    }
});
```
and then 
```
let logger = log4js.getLogger('first');
    logger.addContext("hack", `folder1/${new Date().toDateString()}-hack`);
    logger.info("First Log");


let logger = log4js.getLogger('second');
    logger.addContext("hack", `hack2/${new Date().toDateString()}-hack`);
    logger.info("Second Log");

```

Is there any better way to do the same

**Comment by nomiddlename** [comment]
Not at the moment, no. I'd be happy to see a pull request to implement it, though.

**Comment by cula** [comment]
@AgnibhaBose I've recoded some codes of one dependency of `log4js-node` to support this. So this can be  implemented after some changes in `log4js-node`. Maybe finished with milestone 3.0.0

**Comment by nomiddlename** [comment]
This should be fixed in log4js@4.0.0 in npm (thanks to @cula for the changes) - you should be able to pass any of the file and dateFile options to multiFile now.
